docs/workshop/index.md

```diff
@@ -0,0 +1,410 @@
+                COLOR 254 0 0
+                SYMBOL 'star'
+                SIZE 8
+            END
+        END
+        TEMPLATE 'dummy'
+    END # End of Layer
+END # End of Map
+```
+
+## Add Firestations to the catalog
+
+* GeoMoose separates the difference between the source-data with
+  `<map-source />` es and presentation with the use of the `<catalog>`.
+* In `mapbook.xml` goto line 427 and add the following on the next line:
+    ```xml
+    <layer src="firestations/fire_stations" title="Firestations"/>
+    ```
+* This will add the `fire_stations` layer of the `firestations` source
+  to the catalog with the label "Firestations".
+
+---
+
+* "Hard" Reload the Browser or clear-the-cache and reload.
+  *Pro tip: Chrome can be very aggressive at caching AJAX loaded XML.*
+
+* The catalog show now have a 'Firestations' layer at the top!
+
+![Firestations in the catalog](./images/firestations-in-catalog.png)
+
+## Adding identify to Firestations
+
+* WMS has the GetFeatureInfo request which GeoMoose will use to fetch feature data.
+* For a layer to work with identify it needs to have a `<template>` named `identify`
+* In the `mapbook.xml` file update the `firstations` `<map-source>` definition:
+
+```xml
+<map-source name="firestations" type="mapserver">
+    <file>./demo/firestations/firestations.map</file>
+    <layer name="fire_stations">
+        <template name="identify"><![CDATA[
+        <div class="result-item">
+            <div class="result-title">
+            Firestation
+            </div>
+            <b>Station City:</b> {{ properties.Dak_GIS__4 }}<br>
+            <b>Station Number:</b> {{ properties.Dak_GIS__5 }}<br>
+        </div>
+        ]]></template>
+    </layer>
+</map-source>
+```
+
+## More information on templates
+
+* GeoMoose has a rich template system provided by Mark.up.
+  [More infoformation on GeoMoose templates here.](http://geomoose.github.io/gm3/templates.html)
+* Features in GeoMoose are processed through the templates as
+  [GeoJSON](http://geojson.org). For example:
+
+```javascript
+  {
+    "type": "Feature",
+    "geometry": {
+        "type": "Point",
+        "coordinates": [0, 0]
+    },
+    "properties": {
+        "label": "Null Island",
+        "partyScope": 555
+    }
+  }
+```
+
+## How does identify work?
+
+1. It's included in the `index.html` file:
+    ```html
+    <script type="text/javascript" src="../geomoose/dist/services/identify.js"></script>
+    ```
+
+2. Then registered in the `app.js` file:
+    ```javascript
+    app.registerService('identify', IdentifyService);
+    ```
+
+3. And finally, put on the toolbar in the `mapbook.xml` file.
+    ```xml
+    <tool name="identify" title="Identify" type="service"/>
+    ```
+
+## Learning more about identify and services
+
+* Open `geomoose/dist/services/identify.js` in an editor.
+
+* From the GeoMoose website, services are defined as:
+> A service is used to collect information from the user and the map then
```

Comment:
In the PDF: This didn't quote, it shows the `>` as the same paragraph as the prior text.